src/err.cpp

```diff
@@ -100,128 +100,112 @@ const char *zmq::wsa_error_no (int no_, const char *wsae_wouldblock_string)
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              : (no_
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 == WSANO_DATA)
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  ? "Valid name no data record of requested"
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  : "error not defined";
+    switch(no_) {
```

Comment:
space before the bracket (see: https://travis-ci.org/zeromq/libzmq/jobs/366119755 )

Reply:
Just saw it and commited :)